transport-native-unix-common-tests/src/main/java/io/netty/channel/unix/tests/UnixTestUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.unix.tests;
+
+import io.netty.channel.unix.DomainSocketAddress;
+import io.netty.channel.unix.Socket;
+
+import java.io.File;
+import java.io.IOException;
+
+public final class UnixTestUtils {
+    public static DomainSocketAddress newSocketAddress() {
+        try {
+            File file;
+            do {
+                file = File.createTempFile("NETTY", "UDS");
+                file.delete();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Exceptional return value of java.io.File.delete() ignored in io.netty.channel.unix.tests.UnixTestUtils.newSocketAddress() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=findbugs%3ARV_RETURN_VALUE_IGNORED_BAD_PRACTICE)